### PR TITLE
Euwe-3 announcement

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@cybette got the okay from QE on smoke testing. Please merge when possible.

File: site/_posts/2017-04-17-Announcing-Euwe-3.html.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+[CVE-2017-2653](https://access.redhat.com/security/cve/CVE-2017-2653) - A number of unused delete routes are present in CloudForms which can be accessed via GET requests instead of just POST requests. This could allow an attacker to bypass the protect_from_forgery XSRF protection causing the routes to be used. This attack would require additional cross-site scripting or similar attacks in order to execute.
+This issue was reported by Martin Povolny (Red Hat).
+
+and here are just a few of the things added since Euwe-2 release
+- UI: Add new Subnet must be disabled when there is no cloud provider present
+- Inventory refresh fail on timeout, after ~2 minutes
+- Security groups/Floating IPs not displayed in Network Topology View
+- External authentication works when logging into the Admin UI but doesn't work for the same user to get into the Service UI
+- Unable to bring VM out of retirement from details page
```
Comment (cybette) [inline site/_posts/2017-04-17-Announcing-Euwe-3.html.md]: @JPrause I was wondering where these PRs are from and couldn't find matching ones in the changelog. Then I notice they are titles of CFME bugs. Even though we maintain close downstream/upstream development, let's not mix them for clarity's sake. Perhaps you can use the description in the corresponding ManageIQ PRs/commits, or even directly from the MIQ changelog.

File: site/_data/releases.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  tag: euwe-3
+  filename: euwe-3
 
 prerelease:
- name: Fine-1-Beta1
- tag: fine-1-beta1
- filename: fine-1-beta1
+ name: Fine-1-RC1
+ tag: fine-1-rc1
+ filename: fine-1-rc1
```
Comment (cybette) [inline site/_data/releases.yaml]: I think the changes for Fine Beta1 -> RC1 should be done in #519 instead of this PR which is for Euwe-3.

**Comment by JPrause** [comment]
@cybette fixed both issues. Should be good to go. Please merge, when checks are green.

**Pull request merged**